homeassistant/components/light/wink.py

```diff
@@ -8,12 +8,17 @@
 """
 import logging
 
-from homeassistant.components.light import ATTR_BRIGHTNESS
+from homeassistant.components.light import (ATTR_BRIGHTNESS, ATTR_COLOR_TEMP)
 from homeassistant.components.wink import WinkToggleDevice
 from homeassistant.const import CONF_ACCESS_TOKEN
 
 REQUIREMENTS = ['python-wink==0.4.2']
 
+TEMP_MIN = 1900               # Wink minimum temperature
+TEMP_MAX = 6500               # Wink maximum temperature
+TEMP_MIN_HASS = 154           # home assistant minimum temperature
```

Comment:
Instead of defining them here, could you:
 - extract them as constants into `light/__init__.py` from [this line](https://github.com/balloob/home-assistant/blob/dev/homeassistant/components/light/__init__.py#L234)
 - import them from the light component